File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.java
class Solution {
    public String solution(String s) {
        StringBuilder answer = new StringBuilder();
        int count = 0;
        String[] str = s.split("");
        
        for (String ss : str) {
            if (ss.equals(" ")) {
                count = 0;
            } else {
                ss = (count % 2 == 0) ? ss.toUpperCase() : ss.toLowerCase();
                count++;
            }
            answer.append(ss);
        }
        return answer.toString();
    }
}

File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.py
def solution(s):
    answer = ''
    m = s.split(' ')
    for c in m:
        for k in range(len(c)):
            if k % 2 == 0:
                answer += c[k].upper()
            elif k % 2 != 0:
                answer += c[k].lower()
        answer += ' '
        
    
    return answer[:-1]